File: Week1/problemSet/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // get height
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h > 8 || h < 1);
    int i = 0;

    // print blocks in a loop
    while (i < h)
    {
        int whiteSpaces = h - i - 1;
        int blocks = h - whiteSpaces;
        int j = 0;

        // print first white spaces
        while (j < whiteSpaces)
        {
            printf(" ");
            j++;
        }
        j = 0;

        // print first blocks
        while (j < blocks)
        {
            printf("#");
            j++;
        }
        j = 0;

        // print gap
        printf(" ");
        printf(" ");

        // print second blocks
        while (j < blocks)
        {
            printf("#");
            j++;
        }
        j = 0;

        // orint new line
        printf("\n");
        i++;
    }
}

File: Week6/sentimental-mario-more/mario.py
# TODO

from sys import argv

# get height
while True:
    try:
        height = int(input("Height: "))
        if (height >= 1) and (height <= 8):
            break
    except:
        print("", end="")


# print blocks
for i in range(height):
    whiteSpaces = height - i - 1
    blocks = height - whiteSpaces
    j = 0

    # first whitespaces
    while j < whiteSpaces:
        print(" ", end="")
        j += 1

    j = 0

    # first batch
    while j < blocks:
        print("#", end="")
        j += 1

    j = 0

    # spaces
    print(" ", end="")
    print(" ", end="")

    # second batch
    while j < blocks:
        print("#", end="")
        j += 1

    print()
